include/aspect/material_model/non_dimensional.h

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
Given how you spell the material model's name, I think you should also call the file `nondimensional.h` (without the underscore; same for the `.cc` file of course).